.changeset/heavy-streets-do.md

```diff
@@ -0,0 +1,5 @@
+---
+'@commercetools/composable-commerce-test-data': minor
+---
+
+Adding shipping, item shipping details and item shipping target models
```

Comment:
Could we please are a bit more context to the changeset? 🙏 

[Here](https://github.com/commercetools/test-data/pull/858/files#diff-0956e6b02a0a080c2bcc42964b0c3bf1fbafa88edfcaa3922160fa646df09fad)'s an example of what we've been using for new models.